keps/NNNN-kep-template/README.md

```diff
@@ -452,6 +453,36 @@ You can take a look at one potential example of such test in:
 https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
 -->
 
+### Testing Quality
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+###### What is the current (unit) test coverage of packages that will be touched when implementing this KEP?
+
+<!--
+List the packages where non-boilerplate code will be added/changed with their
+current coverage as reported by test-coverage suite under
+https://testgrid.k8s.io/sig-testing-canaries#ci-kubernetes-coverage-unit
+-->
+
+###### If the coverage is below 80%, what additional tests will be added before implementing this KEP?
+
+<!--
+We need to ensure that all existing areas of the code that will be touched by
+this enhancement are adequatly tested to mitigate the risk of regressions.
+--->
+
+###### Please link the integration and e2e tests that were added/extended for this KEP
+
+<!--
+This question should be filled when targeting Beta release.
+The goal is to ensure that we don't accept enhancements with inadequate testing.
+So even if those hit Alpha in such state, it won't be possible to target Beta
+graduation (and thus by-default enablement) until the testing is sufficient.
+-->
+
```

Comment:
Additionally, as one moves from Beta to GA what are our test expectations? Above you mention 80% for unit tests, what do we expect to see for e2e? In an ideal world, what would e2e on a feature look like to allow it to be targeted to GA? Let's be explicit here.

Reply:
I'd also like to see (as the KEP moves through stages) passage rates for the specific implemented tests (as opposed to the general passage rates) so that we can identify flaky tests.

Reply:
> In an ideal world, what would e2e on a feature look like to allow it to be targeted to GA? Let's be explicit here.

I don't think it's possible to describe it generically here.
Or are you asking just for adding a question about this? If so that makes sense.

> I'd also like to see (as the KEP moves through stages) passage rates for the specific implemented tests (as opposed to the general passage rates) so that we can identify flaky tests.

Good point - let me add that.